client/src/components/Accordions.jsx

```diff
@@ -3,7 +3,9 @@ import {AccordionContainer} from '../styling.jsx';
 
 const Accordions = (props) => {
   return (
-    <AccordionContainer>Accordions go here</ AccordionContainer>
+    <AccordionContainer>
+      Accordions
+    </ AccordionContainer>
```

Comment:
I'm curious about the space after the `</` in `</ AccordionContainer>. Maybe a typo?

Reply:
I think I misused the closing slash space rule for single-tag for this one. My linter isn't very fond of slashes...